tasks/cleanup.yml

```diff
@@ -2,11 +2,15 @@
 - name: Get previous app deploy listing
   command: ls -tr1 chdir={{ react_versioned_path }}
   register: versioned_list
-  changed_when: "versioned_list.stdout_lines|length > 1"
+  changed_when: false
```

Comment:
Why this change?

Reply:
This task is setting a fact, it will always report as changed on any run.

Reply:
cool